js/viz/docs/docObjectsDxSankey.js

```diff
@@ -0,0 +1,103 @@
+
+/**
+* @name dxSankeyNodeItem
+* @publicName NodeItem
+*/
+var NodeItem = {
+    /**
+    * @name dxSankeyNodeItemfields_title
+    * @publicName title
+    * @type string
+    */
+    title: undefined,
+    /**
+    * @name dxSankeyNodeItemfields_linksIn
+    * @publicName linksIn
+    * @type Array<Object>
+    */
+    linksIn: undefined,
+    /**
+    * @name dxSankeyNodeItemfields_linksOut
+    * @publicName linksOut
+    * @type Array<Object>
+    */
+    linksOut: undefined,
+    /**
+    * @name dxSankeyNodeItemmethods_hover
+    * @publicName hover(state)
+    * @param1 state:boolean
+    */
+    hover: function () { },
+    /**
+    * @name dxSankeyNodeItemmethods_ishovered
+    * @publicName isHovered()
+    * @return boolean
+    */
+    isHovered: function () { },
+    /**
+    * @name dxSankeyNodeItemmethods_showtooltip
+    * @publicName showTooltip()
+    */
+    showTooltip: function(){ }
+};
+
+/**
+* @name dxSankeyLinkItem
+* @publicName LinkItem
+*/
+var LinkItem = {
+    /**
+    * @name dxSankeyLinkItemfields_connection
```

Comment:
As I see from the code, this object always contains three fields: `from`, `to`, and `weight`. It would be good to create a separate entity for it. We already have a similar case: [entity declaration](https://github.com/DevExpress/DevExtreme/blob/8fb7a7f223d0e66c2784fe377cca49809456c76a/js/viz/docs/docObjectsDxChart.js#L21) | [reference example](https://github.com/DevExpress/DevExtreme/blob/8fb7a7f223d0e66c2784fe377cca49809456c76a/js/viz/docs/docObjectsDxChart.js#L57).